### ISSUE TITLE
FocusMode not saving its own window size

### ISSUE THREAD

**Issue by eugenov** [kingToolbox/WindTerm#205]
WindTerm 2.2.0 Prerelease 1
Hello.
FocusMode not saving its own window size. At startup WindTerm restore last window size of Normal mode, even when focus mode set to default.

**Comment by kingToolbox** [comment]
Yes, this is a known issue. At present, to restore the `focus mode` after restarting is to first restore the normal mode window, and then automatically enter the `focus mode` according to the configuration of `focus mode`. If you added or hidden panes or changed the window size after entering `focus mode` last time, it will not be restored. 

The reason for this design is that once you still need to switch from `focus mode` to `normal mode`, the window size and state of the last normal mode have to be restored. So currently, only the window size in normal mode is saved before exiting. 

**Saving and restoring the window size of both modes will complicate the design. I have tried it once during development, but I failed.** I will consider making improvements in subsequent versions and update the progress here, thank you.

**Comment by eugenov** [comment]
Thanks for clearing, I don't mind to live with that. You could close this issue then.

**Comment by eugenov** [comment]
It will be ok for me if window size and position will remain the same for focus mode and for normal mode. I mean when you switch to normal mode window wont change, just panes will appear. How about that? Now changing modes results in unpredictable moving/resizing window - it is not too convenient.

**Comment by eugenov** [comment]
Closing with 2.4.0 prerelease update
